src/anchorfm-pupeteer/index.js

```diff
@@ -109,7 +110,7 @@ async function postEpisode(youtubeVideoInfo) {
   async function openNewPage(url) {
     const newPage = await browser.newPage();
     await newPage.goto(url);
-    await newPage.setViewport({ width: 1600, height: 789 });
+    await newPage.setViewport({ width: 1600, height: 1600 });
```

Comment:
why are you doing this?

Reply:
The viewport height is enlarged to get better screenshot covering more area.

Reply:
Could we set a real device viewport? Like a 2k resolution would be enough I believe

Reply:
So for example, 2k reeolution is sokething like 2560 x 1440? 

If that's so, I love the idea, want to make 2560 width and 1440 height?

Reply:
Yes, let's go for it

Reply:
What Ivwill also do is add back the logging when error happens. This is to log if error happens before puppeteer code starts.

Reply:
Done, I made the changes.